src/styles/global.css

```diff
@@ -53,9 +53,10 @@ a, a:hover, a:active, .page-link {
     background: #aaa;
 }
 
-.mapSelected {
-    stroke:#421B49;
-    stroke-width:4;
+/* Style focused <path> elements or selected state */
+path:focus, .mapSelected {
```

Comment:
This fixes focus not being visible when you tab through the map, but the map's DOM nodes are in A-Z order meaning it looks _absurd_ and is still inaccessible, but this is still an improvement:


https://user-images.githubusercontent.com/3187531/158079668-42f9393b-1f61-4f27-95a5-061b58d35f05.mp4

Reply:
looks good to me. we will likely be swapping this map out for just an SVG of Illinois, so it may be a non-issue. does help with the home page map, though!